decentraland/unity-renderer
# refactor: refactor of PlayerInfoHUD adding more test coverage

## What does this PR change?

- Defined better responsabilities between view<->controller
- Controller now injects collaborators to be able to mock dependencies and isolate automatization tests
- Fixed non-working `CreateCollectibles` test (was throwing different test results depending if runned individually or not)

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=refactor/player-info-card
2. Click on any user avatar to check its info
3. All card elements should work properly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
